src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -394,13 +392,21 @@
     <!-- Non-null type is required for this node kind -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
 
+    <!-- Various assignable expressions, including some BoundDeconstructionVariables for nested variables -->
     <Field Name="LeftVariables" Type="ImmutableArray&lt;BoundExpression&gt;"/>
```

Comment:
Ok, given the current implementation, it looks like LeftVariable is always a flat list of targets, BoundDeconstructionVariables is never among them. Please adjust the comment accordingly. 

---
In reply to: [65778993](https://github.com/dotnet/roslyn/pull/11715#discussion_r65778993) [](ancestors = 65778993,65776149)